### PR TITLE
Gitignore Updates

### PR DESCRIPTION
**Pull request by ghickman**
A collection of gitignore house keeping spurred by the switch to enabling WAL mode in SQLite.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,18 +1,12 @@
-*~
-.#*
-*#
-model.log
-*/input.csv
-__pycache__
-.python-version
-*sqlite3
+*sqlite3*
```
Comment (inglesp) [inline .gitignore:1]: This looks a little bit too broad.  (I've got a file a project called `sqlite_wrapper.py`, which would almost get caught by this.)  Could we put the database files in a directory, and add that directory to .gitignore?

File: .gitignore [hunk 2]
```diff
@@ -1,18 +1,12 @@
...
 .env
+.envrc
 .idea/
-node_modules/
 /assets/dist
 /assets/stats.html
 /coverage
+htmlcov
+node_modules/
+releases
```
Comment (inglesp) [inline .gitignore:11]: This'll confuse us if we extract the releases into their own app.

**Review by inglesp — APPROVED** [review]
Probably fine, but a couple of paranoid comments.

File: .gitignore (hunk 2 above)
Comment (ghickman) [reply .gitignore:11]: Future us problems 🙂

File: .gitignore (hunk 1 above)
Comment (ghickman) [reply .gitignore:1]: Good point 👍

**Pull request merged**